### PR TITLE
Soham Ratnaparkhi

### PR DESCRIPTION
**Pull request by SohamRatnaparkhi**
Hello, I am **Soham Ratnaparkhi** and I intend to start my open-source contributions journey through LinkFree. Thank you!

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1269"></a>

### CODE REVIEW

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
There’s a failing test due to an invalid character in line 20, if you could fix it and push your changes this should be good to go.

File: public/data/SohamRatnaparkhi.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SohamR_7113",
+      "icon": "twitter"
+    },
+    {
+      "name": "My linkdIn profile!",
+      "url": "https://www.linkedin.com/in/soham-ratnaparkhi-3a3775ab/",
+      "icon": "LinkedIn	"
```
Comment (kumarsonsoff3) [inline public/data/SohamRatnaparkhi.json:20]:
```suggestion
      "icon": "LinkedIn"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Minor fixes required!

**Review by Vyvy-vi — COMMENTED** [review]
There was a duplicate PR for the same, that was merged a few days ago (#1271)
We're closing this PR, as your profile has been added to LinkFree! - http://linkfree.eddiehub.org/SohamRatnaparkhi

**Pull request closed without merging**